### DISCUSSION TITLE
Please my home page can not show my followers's stars now!!!

### DISCUSSION THREAD

**Discussion by lucasjinreal** [community/community#66676] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Please my home page can not show my followers's stars now!!!

@github

**Comment by SinclairCoder** [comment]
I also want the original page!!
I want to see which repositories the people I follow star!!!

**Comment by Bian-Sh** [comment]
This must be a bug, as I toggled the star filter, but now I can't see any of my followers' starred repositories.

**Comment by misaraadel** [comment]
I think will appear in setting
check your setting

**Reply by Bian-Sh** [reply]
how , pls ？

**Reply by lucasjinreal** [reply]
it has nothing to do with settings, and it just effect some certain people, my another account can show normally........

**Comment by thecmdrunner** [comment]
We still have it here (FOR NOW):
https://github.com/dashboard-feed

**Comment by ettaboyle** [comment]
_We've added the following update to the [Updates to your GitHub Feed](https://github.com/orgs/community/discussions/65343#top) Discussion Announcement originally posted August 29, 2003._


## _Sept. 12th Update_
_With the latest feed update on September 6, 2023, we made changes to the underlying technology of the feed to improve overall platform performance. As a result, we removed the functionality for “push events for repositories a user is subscribed to”. We don’t take these changes lightly, but as our community continues to grow tremendously, we have to prioritize our availability, user experience, and performance._ 


_Thanks to feedback from the community, we have updated our [changelog](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/) to clarify and have fixed the following bugs from the initial September 6th release:_


- _We were showing releases for organizations that you follow, instead of only showing them for repositories that you watch or star. This is now fixed._ 
- _We have also addressed a problem where 'Followed You' cards were sometimes appearing out of chronological order._
- _We do not plan to re-include “push events for repositories a user is subscribed to” for the reasons listed above, but we will continue to address feedback as it aligns with our platform goals._


_We understand that many of you are upset with the recent changes to your feed. We should have done a better job communicating recent changes and how those decisions relate to our broader platform goals. Your continued feedback is invaluable as we evolve and continue to strive to provide a first-class developer experience that helps every developer be happier and more productive._

**Reply by lucasjinreal** [reply]
I have to say, this is not a feature, it's a bug, please admit it and please fire the product manger at the moment. @GitHub this new really like a shit (not inferface but functionality)

**Reply by linonetwo** [reply]
Still not working.

**Comment by lucasjinreal** [comment]
@ettaboyle When will stars back?

<img width="1069" alt="image" src="https://github.com/community/community/assets/21303438/2f46a3d7-bb43-4e42-b287-27e51c09ac08">

this old page even has no padding...

**Comment by sooftware** [comment]
Please old feed back..

**Comment by CavalcanteLeo** [comment]
I hate with all my strength this new timeline feed, the worst thing GitHub has ever done.

It's not pleasant and enjoyable to browse the feed anymore, to discover new repos, to see commits. this is disgusting now 

![ae8d7d4a0661c9ce7101c5f863ca895a](https://github.com/community/community/assets/1735944/fd144c81-81f8-459c-bf8e-a28c589f36ff)

**Reply by lucasjinreal** [reply]
@github please make a useful product, not make users disgusting!

**Comment by dy** [comment]
New feed is fine, but all stars from the feed are gone. That was the main reason for me to come back to @github regularly to just doomscroll in order to find some fresh cool stuff. Now it's not as social anymore. Was that a planned change @github?
https://github.com/dashboard-feed is useful, but I am seriously searching for alternative platforms / communities now to share/discover code. And I am not the first one.
Is there an underlying plan to kill the community?

**Comment by developer-sharib-804** [comment]
my followers not show why

**Comment by dmarx** [comment]
this is a thing again. please fix the stars feed. I'm not seeing any activity from people i'm following at all, just the activity of people starring my repos. I don't care when people star my repos, I want to see what the people I am following are starring.

**Comment by LiteBrite82** [comment]
This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.

**Reply by Enigmatisms** [reply]
So how do we let the Github employees know that some new features suck? This is clearly a bug existing for a long time. It shows people starring my repo or following me, every day even if for almost two weeks I have no new follower and repo stargazer. It just won't work properly. Aren't **discussion** supposed to be a place for **feedback** since we can get here by clicking **send feedback**? Do they really care about our feedback?
